.github/CONTRIBUTING.md

```diff
@@ -1,93 +1,150 @@
+
+Good
+
+> Fix installation method
+
+> What's the new behaviour?
+> 
+> * Dependencies updated
+> * Documentation improved
+
+Also, avoid using your branch or the commit guidelines to name your PR, for example:
+
+Bad
+
+> feat(private): implement private data method
+
+Good
+
+> Feature private information
 
-### Type
+In case your contribution has to do with reports, remember those are created in the develop branch, nor master or PR's.
 
-Must be one of the following:
+* If we suggest changes then:
 
-* **feat**: A new feature
-* **fix**: A bug fix
-* **docs**: Documentation only changes
-* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
-  semi-colons, etc)
-* **refactor**: A code change that neither fixes a bug or adds a feature
-* **test**: Adding missing tests
-* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
-  generation
+  * Make the required updates.
 
-### Scope
+  * Rebase your branch and force push to your GitHub repository (this will update your Pull Request):
 
-The scope could be anything specifying place of the commit change. For example `app`,
-`gen`, `docs`, `gen:view`, `gen:route`, `gen:service`, etc... It is preferred to use ascope matching an available label for issues. If no label match the scope of your PR, please, get in touch with us.
+    **That's it! :tada:  Thank you for your contribution!**
 
-### Subject
+#### After your pull request is merged
 
-The subject contains succinct description of the change:
+You can safely delete your branch and pull the changes from the main (upstream) repository:
 
-* use the imperative, present tense: "change" not "changed" nor "changes"
-* don't capitalize first letter
-* no dot (.) at the end
+* Delete the remote branch on GitHub either through the GitHub web UI or your local shell as you prefer.
 
-### Body
+## Coding Rules
 
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes"
-The body should include the motivation for the change and contrast this with previous behavior.
+To ensure consistency throughout the source code, keep these rules in mind as you are working:
 
-### Footer
+* All features or bug fixes must be [tested](#tests) by one or more specs (unit-tests).
+* All methods must be documented.
+* This project follows the [GLPI development guidelines](https://readthedocs.org/projects/glpi-developer-documentation/).
 
-The footer should contain any information about **Breaking Changes** and is also the place to
-reference GitHub issues that this commit **Closes**.
+## Tests
 
-A detailed explanation can be found in this [document][commit-message-format].
+* Make sure test-suite passes: [atoum](http://atoum.org/)
+* Run:
 
-[commit-message-format]: https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y
+```console
+  npm test
```

Comment:
For testing the correct commands are:

  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-install -ncc
  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-unit --nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI
  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-integration --nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI
  - vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-uninstall --ncc
  - vendor/bin/phpcs -p --standard=vendor/glpi-project/coding-standard/GlpiStandard/ *.php install/ inc/ front/ ajax/ tests/; 

Or you can referrer to "script" section on .travis.yml file for more details.

Reply:
Thanks, I'll make the changes

Reply:
Why not provide some scripts for windows and linux to ease unit tests ?